android/src/main/java/com/urbanairship/reactnative/AirshipHeadlessEventService.kt

```diff
@@ -11,7 +11,7 @@ import com.urbanairship.android.framework.proxy.ProxyLogger
 
 class AirshipHeadlessEventService : HeadlessJsTaskService() {
 
-    override fun getTaskConfig(intent: Intent): HeadlessJsTaskConfig? {
+    override fun getTaskConfig(intent: Intent?): HeadlessJsTaskConfig? {
```

Comment:
RN converted the java class to kotlin and marked it as optional. This change could be backported if we wanted.